ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-maistra-2.4.yaml

```diff
@@ -0,0 +1,20 @@
+build_root:
```

Comment:
why do we have these?

I thought we were going to add only tide

Reply:
@jwendell I removed those ci-config yamls. Yes only need tide so far.